core/matrix/dense.cpp

```diff
@@ -269,6 +334,26 @@ void Dense<ValueType>::sub_scaled_impl(const LinOp* alpha, const LinOp* b)
 }
 
 
+template <typename ValueType>
+void Dense<ValueType>::compute_dot(const LinOp* b, LinOp* result,
+                                   Array<char>& tmp) const
+{
+    GKO_ASSERT_EQUAL_DIMENSIONS(this, b);
+    GKO_ASSERT_EQUAL_DIMENSIONS(result, dim<2>(1, this->get_size()[1]));
+    auto exec = this->get_executor();
+    if (tmp.get_executor() != exec) {
+        tmp.clear();
+        tmp.set_executor(exec);
+    }
+    auto local_b = make_temporary_clone(exec, b);
+    auto local_res = make_temporary_clone(exec, result);
+    auto dense_b = make_temporary_conversion<ValueType>(local_b.get());
+    auto dense_res = make_temporary_conversion<ValueType>(local_res.get());
+    exec->run(
+        dense::make_compute_dot(this, dense_b.get(), dense_res.get(), tmp));
```

Comment:
`make_compute_dot_dispatch` ? Same for other functions.

Reply:
is it missing dispatch?

Reply:
no, the corresponding operation calls _dispatch, see the beginning of the file. Do you think I should change it here as well? I wanted to keep the diff small, but that of course may create some confusion. I'd assume once #991 is executed and merged, we could probably reverse this?

Reply:
I see. I do not notice the function is point to the dispatch one.
it should be okay